docker/models/images.py

```diff
@@ -395,7 +395,7 @@ def load(self, data):
 
         return [self.get(i) for i in images]
 
-    def pull(self, repository, tag=None, **kwargs):
+    def pull(self, repository, tag='latest', **kwargs):
         """
         Pull an image of the given name and return it. Similar to the
         ``docker pull`` command.
```

Comment:
The wording of the docstring should be updated to make it more clear. E.g.:
```
If no tag is specified, all tags from that repository will be pulled.
```
should be
```
If tag is None, all tags from that repository will be pulled.
```